crates/wash-cli/src/config.rs

```diff
@@ -89,9 +103,19 @@ async fn put_config(
     sp.finish_and_clear();
 
     let message = if config_response.message.is_empty() && config_response.success {
-        format!("Configuration {name} put successfully.")
+        let mut out_name = name;
+        let mut config_type = "Configuration";
+        if is_secret {
+            out_name = out_name
+                .strip_prefix(format!("{SECRET_PREFIX}_").as_str())
+                .unwrap_or(name);
+            config_type = "Secret";
```

Comment:
supernit: Should this be a secret reference to be clear that it's not a secret?

Reply:
I actually don't feel strongly about this, it's clear enough from our help texts